.github/workflows/base/ci.yml

```
@@ -0,0 +1,334 @@
+name: Base CI Workflow (Shared Patterns)
+
+on:
+  workflow_call:
+    inputs:
+      project-type:
```

actionlint (1.7.11)

[error] "type" is missing at "project-type" input of workflow_call event

(syntax-check)